go-controller/pkg/config/config.go

```diff
@@ -123,6 +123,7 @@ type KubernetesConfig struct {
 	Token              string `gcfg:"token"`
 	ServiceCIDR        string `gcfg:"service-cidr"`
 	OVNConfigNamespace string `gcfg:"ovn-config-namespace"`
+	OVNKubePodIP       string `gcfg:"ovnkube-pod-ip"`
```

Comment:
@numansiddique let's just do "PodIP" as OVNKube is a bit redundant.

Reply:
Done